app/src/main/java/net/skyscanner/backpack/demo/compose/BottomSheetStory.kt

```diff
@@ -106,8 +107,30 @@ fun ImageBottomSheetStory(
 @Composable
 @BottomSheetComponent
 @ComposeStory(name = "Modal")
-fun ModalBottomSheetStory(
+fun ModalBottomSheetNoTopBarStory(
     modifier: Modifier = Modifier,
+) {
+    ModalBottomSheetStory(modifier = modifier)
+}
+
+@Composable
+@BottomSheetComponent
+@ComposeStory(kind = StoryKind.DemoOnly, name = "Modal with TopBar")
+fun ModalBottomSheetWithTopBarStory(
+    modifier: Modifier = Modifier,
+) {
+    ModalBottomSheetStory(
+        modifier = modifier,
+        title = stringResource(id = R.string.modal_bottom_sheet_title),
+        isClosable = BpkModalBottomSheetCloseAction.Default(stringResource(id = R.string.navigation_close)),
+    )
+}
+
+@Composable
+internal fun ModalBottomSheetStory(
+    modifier: Modifier = Modifier,
+    title: String? = null,
+    isClosable: BpkModalBottomSheetCloseAction = BpkModalBottomSheetCloseAction.None,
```

Comment:
i think the api table means that `isClosable` should be a boolean. but yes we can enforce the "needs to be set if `isClosable=true` by making this an enum. i would name this `closeButton`, though to align with the API table and be less confusing than a boolean name which really is an enum